packages/cli/src/commands/model/list.ts

```diff
@@ -74,14 +74,23 @@ export default class ModelList extends BaseCommand<ModelListFlags> {
     }
   }
 
-  getFieldsFromEdges(ceramicIndexer: CeramicClient, edges: Array<Edge<StreamState>>): Array<PartialModelDefinition> {
+  getFieldsFromEdges(ceramicIndexer: CeramicClient, edges: Array<Edge<StreamState | null>>): Array<PartialModelDefinition> {
     return edges.map((edge) => {
-      const stream = ceramicIndexer.buildStreamFromState(edge.node)
-      return {
-        id: stream.id.toString(),
-        name: (stream.content as Record<string, any>).name as string,
-        description: (stream.content as Record<string, any>).description as string,
+      if (edge && edge.node) {
+        const stream = ceramicIndexer.buildStreamFromState(edge.node)
+        return {
+          id: stream.id.toString(),
+          name: (stream.content as Record<string, any>).name as string,
+          description: (stream.content as Record<string, any>).description as string,
+        }
+      } else {
+        return {
```

Comment:
Why empty contents here please? Let's filter out null results rather than returning them please.

Reply:
Are you ok with pagination working in unexpected ways if we filter out empty results, @PaulLeCam

Reply:
How does it affect pagination? We can always perform multiple queries to fill a page if needed, IMO it's better than empty rows.

Reply:
I guess you answered your own question about the pagination :). Cool, thanks!